### ISSUE TITLE
Why not hide `data_dir` folder?

### ISSUE THREAD

**Issue by escorciav** [epic-kitchens/epic-lib#83]
Is there a particular reason for which the default `data_dir` isn't a hidden folder?

- It seems a bit intrusive to create a visible folder in the $HOME of the user. Other libraries and apps (PyTorch, Jupyter, etc.) use the hidden folder pattern.

- I do appreciate that we can set the `data_dir`.

BTW, thanks for providing this library to easily interact with the dataset.

**Comment by willprice** [comment]
Ahhh... yes, this is an accident, I meant to default to the standard XDG_CACHE_DIR path, but forgot to append `.cache` to the path:

```python
        if local_dir is None:
            cache_dir = Path(os.environ.get("XDG_CACHE_HOME", Path.home()))
            local_dir = cache_dir / "epic_kitchens" / version
```

**Comment by escorciav** [comment]
Thanks! please consider it for next release.

(out of scope of issue) btw, I noticed that some functions like `meta.action_id_from_verb_noun` do not appear in the documentation.

**Comment by willprice** [comment]
> (out of the scope of issue) btw I noticed that some functions, like meta.action_id_from_verb_noun, do not appear in the documentation.

Are you sure?
https://epic-kitchens.readthedocs.io/en/latest/epic_kitchens.html#epic_kitchens.meta.action_id_from_verb_noun

**Comment by willprice** [comment]
Should be fixed in v1.7.1
